oneview/data_source_server_profile.go

```diff
@@ -617,13 +627,13 @@ func dataSourceServerProfile() *schema.Resource {
 				ForceNew: true,
 			},
 			"management_processor": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
Can you use TypeSet instead of TypeList?

Reply:
fixed